pyasic/web/epic.py

```diff
@@ -0,0 +1,117 @@
+# ------------------------------------------------------------------------------
+#  Copyright 2022 Upstream Data Inc                                            -
+#                                                                              -
+#  Licensed under the Apache License, Version 2.0 (the "License");             -
+#  you may not use this file except in compliance with the License.            -
+#  You may obtain a copy of the License at                                     -
+#                                                                              -
+#      http://www.apache.org/licenses/LICENSE-2.0                              -
+#                                                                              -
+#  Unless required by applicable law or agreed to in writing, software         -
+#  distributed under the License is distributed on an "AS IS" BASIS,           -
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.    -
+#  See the License for the specific language governing permissions and         -
+#  limitations under the License.                                              -
+# ------------------------------------------------------------------------------
+import json
+import warnings
+from typing import Union
+
+import httpx
+
+from pyasic import settings
+from pyasic.web import BaseWebAPI
+
+
+class ePICWebAPI(BaseWebAPI):
+    def __init__(self, ip: str) -> None:
+        super().__init__(ip)
+        self.username = "root"
+        self.pwd = settings.get("default_epic_password", "letmein")
+        self.token = None
+
+    async def send_command(
+        self,
+        command: Union[str, bytes],
+        ignore_errors: bool = False,
+        allow_warning: bool = True,
+        **parameters: Union[str, int, bool],
+    ) -> dict:
+        async with httpx.AsyncClient() as client:
+            is_get = False
+            for i in range(settings.get("get_data_retries", 1)):
+                try:
+                    if parameters.get("post"):
+                        parameters.pop("post")
+                        response = await client.post(
```

Comment:
Could pass `self.pwd` into the command here instead so you don't have to pass it as part of the command, since anytime it is a post request that is a command that will require a password.